File: solutions/7kyu/Isograms/isograms.rb
def is_isogram(string)
  string.downcase.chars.map { |letter| string.downcase.count(letter) > 1 ? '*' : '' }.join.empty?
end

p is_isogram("Dermatoglyphics") #true
p is_isogram("dniybpwyyohbb") #false
p is_isogram("") #true

File: solutions/7kyu/Isograms/isograms.js
function isIsogram(str) {
  if (
    str
      .toLowerCase()
      .split("")
      .sort()
      .join("")
      .match(/(.)\1+/g)
  ) {
    return false;
  } else {
    return true;
  }
}

console.log(isIsogram("Dermatoglyphics")); //true
console.log(isIsogram("aBa")); //false
console.log(isIsogram("")); //true
